packages/components/src/components/data-list/index.js

```diff
@@ -1,5 +1,5 @@
-import DataList from './data-list.jsx';
```

Comment:
we can change this file's extension to .ts as well

Reply:
done